### PR TITLE
[#155861133] Feature room deletion script

### PR DESCRIPTION
**Pull request by mpope9**
Python script to create 5k rooms.  Python script to delete group messages and DMs

@aghchan 
@RuoanJi

### CODE REVIEW

File: delete_rooms.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+from_ctl = subprocess.Popen(['./sbin/ejabberdctl', 'muc_online_rooms', 'global'], stdout=subprocess.PIPE)
+all_rooms = subprocess.Popen(['grep', '-'], stdin = from_ctl.stdout, stdout = subprocess.PIPE)
+from_ctl.stdout.close()
+
+muc_rooms, stderr = all_rooms.communicate()
+
+# Loop through all muc rooms and delete them.
+for muc in muc_rooms.split(os.linesep):
+    name_service = muc.replace('@', ' ')
+    name_service_list = name_service.split(' ')
```
Comment (aghchan) [inline delete_rooms.py]: can you comment the format of the mucJid and does the name contain more than one `@`? if not you can just split on the `@`

Comment (mpope9) [reply delete_rooms.py]: Oh good call, will change

**Comment by aghchan** [comment]
👌

**Comment by RuoanJi** [comment]
the script looks good to me 👍 

not sure if `ejabberdctl` has some kind of limitations if you run commands too many too fast

**Comment by mpope9** [comment]
You were right @RuoanJi, had to add sleep.

**Pull request merged**